mcs/class/corlib/Test/System.Text/EncodingTest.cs

```diff
@@ -128,5 +130,32 @@ public void EncodingName ()
 		{
 			Assert.AreEqual ("Unicode (UTF-8)", Encoding.UTF8.EncodingName);
 		}
+
+		[Test] // https://github.com/mono/mono/issues/11663
+		[Category ("MobileNotWorking")] // Custom Encodings are not included by default on mobile platforms
```

Comment:
this one should work fine on mobile, at least according to the master builds. it only uses ASCII encoding which is available everywhere.